changelogs/fragments/49235-docker_swarm_service-user-default.yaml

```diff
@@ -0,0 +1,5 @@
+---
+bugfixes:
+  - 'fix: fails because of default "user: root" (https://github.com/ansible/ansible/issues/49199)'
```

Comment:
You should add the module name here. Otherwise nobody will know what this refers to (this will simply show up in the changelog without any context).

Reply:
thank you, I added the module name